src/pages/PolicyDetail/__tests__/PolicyDetail.test.tsx

```diff
@@ -345,9 +345,9 @@ describe('src/Pages/PolicyDetail/PolicyDetail', () => {
         expect(screen.getByDisplayValue(/Copy of Not arch x86_64/i)).toBeVisible();
     });
 
-    it('Duplicates navigates to the new policy url', async () => {
+    it.skip('Duplicates navigates to the new policy url', async () => {
```

Comment:
At least these two `skip`ed tests seem to be fine when running the tests locally. Only the provider test seems to fail. Are they flaky or can we not skip the?

Reply:
I tried again, but they are flaky on my device

`npm run test -- policydetail.test`

1st run
> Test Suites: 1 failed, 1 total
> Tests:       1 failed, 26 passed, 27 total

2nd run
> Test Suites: 1 failed, 1 total
> Tests:       2 failed, 25 passed, 27 total

3rd run
> Test Suites: 1 failed, 1 total
> Tests:       1 failed, 26 passed, 27 total

4th run
> Test Suites: 1 failed, 1 total
> Tests:       1 failed, 26 passed, 27 total

5th run:
> Test Suites: 1 failed, 1 total
> Tests:       2 failed, 25 passed, 27 total